pkg/logs/input/file/tailer_key.go

```diff
@@ -0,0 +1,19 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package file
+
+import "fmt"
+
+// tailerKey can be implemented by File and Tailer
+type tailerKey interface {
+	getPath() string
+	getSourceIdentifier() string
+}
+
+// buildTailerKey returns a tailer identifier taking into account the file path and the container ID
+func buildTailerKey(obj tailerKey) string {
```

Comment:
I'm not sure I understood the point, are you saying we can drop the `<path>/` prefix?

Reply:
I'm wondering what would happen in the case of Tailer (or File) returning an empty string, because that branch exist: https://github.com/DataDog/datadog-agent/pull/5745/files#diff-de8b9f30f737ed4f550714585cacee54R52
It will probably never happen if the NewTailer and NewFile constructor is properly used everywhere.

Reply:
yes in that case we will end up with a `<path>/` prefix, similar to what we had before this PR

Reply:
it will reproduce the same behaviour that this PR tries to fix